src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -112,14 +113,16 @@ maybe_create_table() ->
 metrics_module_config(Args) ->
     case os:getenv("CI") of
         "true" ->
-            ejabberd_config:add_local_option(google_analytics_tracking_id, ?TRACKING_ID_CI),
+            ejabberd_config:add_local_option(dev_google_analytics_tracking_id, ?TRACKING_ID_CI),
```

Comment:
I think that the previous name is better suite for this value.

Reply:
In this PR google_analytics_tacking_id is the one that can be configured by the user. The dev in the name correlates with the tracking ID name in Google Analytics (DevStats). In fact metrics that can be seen in that tracking ID are to be used for dev purposes. We could either revert back and find appropriate name for the one that can be configured by the user. The names that I came up with didn't sound right and I thought that this change would be more accurate.

Reply:
To improve future maintenance I think we could even match the name in Google Analytics 1:1 with the name of this option.

Reply:
Right, naming things is often difficult :) I'm for reverting back and using name like `extra_` or `user_google_analytics_tracking_id` for the configurable, additional tracking id.

Reply:
changed to use extra_ prefix and reverted back the rest